account_payment_return/models/payment_return.py

```diff
@@ -0,0 +1,203 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (c) 2011-2012 7 i TRIA <http://www.7itria.cat>
+#    Copyright (c) 2011-2012 Avanzosc <http://www.avanzosc.com>
+#    Copyright (c) 2013 Serv. Tecnol. Avanzados <http://www.serviciosbaeza.com>
+#                       Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+#    Copyright (c) 2014 initOS GmbH & Co. KG <http://initos.com/>
+#                       Markus Schneider <markus.schneider at initos.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning as UserError
+import openerp.addons.decimal_precision as dp
+
+
+class PaymentReturn(models.Model):
+    _name = "payment.return"
+    _inherit = ['mail.thread']
+    _description = 'Payment return'
+    _order = 'date DESC, id DESC'
+
+    company_id = fields.Many2one(
+        'res.company', 'Company', required=True,
+        help="Company",
+        states={'done': [('readonly', True)],
+                'cancelled': [('readonly', True)]},
+        default=lambda self: self.env['res.company']._company_default_get(
+            'account'))
+    date = fields.Date(
+        string='Return date',
+        help="This date will be used as the date for the account entry.",
```

Comment:
s/This date will be used as the date for the account entry
/This date will be used as accounting entry date